File: backend/app/schemas/document.py
from datetime import datetime
from uuid import UUID

from pydantic import BaseModel, ConfigDict

class UploadResponse(BaseModel):
    model_config = ConfigDict(from_attributes=True)

    id: UUID

    filename: str

    content_type: str

    size: int

    message: str = "Document uploaded successfully."

File: frontend/src/api/models/document.ts
// export interface DocumentUploadResponse {
//   id: string;
//   file_name: string;
//   status: string;
// }

export interface DocumentUploadResponse {
  id: string;
  filename: string;
  contentType: string;
  size: number;
  status: string;
  createdAt: string;
  message: string;
}
